experimentA/roi_table.py

```diff
@@ -0,0 +1,113 @@
+import os
+import omero.cli
+import pandas
+from omero.gateway import BlitzGateway
+from omero_metadata.populate import ParsingContext
+
+
+PROJECT = "idr0062-blin-nuclearsegmentation/experimentA"
+TABLE_NAME = "features"
+
+
+def get_features_table(dataset):
+    ann = next(dataset.listAnnotations())
+    file_path = os.path.join("/tmp", ann.getFile().getName())
+
+    with open(str(file_path), 'wb') as f:
+        for chunk in ann.getFileInChunks():
+            f.write(chunk)
+
+    df = pandas.read_csv(file_path, delimiter="\t")
+    return df
+
+
+def iter_rois(roi_service, dataset):
+    for image in dataset.listChildren():
+        result = roi_service.findByImage(image.getId(), None)
+        for roi in result.rois:
+            yield image, roi
+
+
+def delete_tables(conn, dataset):
+    for ann in dataset.listAnnotations(ns="openmicroscopy.org/omero/bulk_annotations"):
```

Comment:
Discussed briefly this morning, we might want to review and agree on which namespace we want to use for these features tables.

- `openmicroscopy.org/omero/bulk_annotations` is the default one in the metadata population plugin (non configurable at the moment)
- in `idr0123`, we used a namespace provided by the 4DN consortium - see https://github.com/IDR/idr0123-mota-mifish/blob/0b83134064330e8ad2e36a8dd3eec2bc0c1ab279/scripts/bed_to_points.py#L37-L39
- some historical studies like `idr0001` have captured features tables (like http://idr.openmicroscopy.org/webclient/omero_table/5223331/ under http://idr.openmicroscopy.org/webclient/?show=screen-3) under a custom test namespace

I suspect the decision should be primarily informed by the type of queries we would expect client would make to access this tabular data.